### PR TITLE
DIV-5373 Respondent landing page for DaRequested

### PR DESCRIPTION
**Pull request by CoffeeOwl97**
DIV-5373 - added show flag to codeceptjs

latest changes

Increase test coverage of switch statement

HTML validation retry request

update content

add dynamic text

# Description

Setting up respondents landing page for when applying for decree absolute.
Includes bug fix for redirection to respondents landing page getting redirected to applicants landing page.

Fixes # (issue)
[DIV-5373](https://tools.hmcts.net/jira/browse/DIV-5373)

## Type of change

Please delete options that are not relevant.
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?
Unit/functional/etc tests.

**Test Configuration**:

* Hardware:
* O/S and version:
* JDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by CoffeeOwl97** [comment]
Formerlly https://github.com/hmcts/div-decree-absolute-frontend/pull/61 brought over with no code changes in order to get a green pipeline build.

File: helpers/caseOrchestrationHelper.js [hunk 1]
```diff
@@ -54,16 +54,21 @@ const formatSessionForSubmit = req => {
 const validateResponse = (req, response) => {
   const { idam } = req;
 
-  const userIsRespondentAndDivorceNotGranted = (idam.userDetails.email === response.data.respEmailAddress) && !(response.state === 'DivorceGranted');
+  const stateIsAwaitingDA = response.state === 'AwaitingDecreeAbsolute';
+  // eslint-disable-next-line max-len
```
Comment (yarpo) [inline helpers/caseOrchestrationHelper.js]: what is max-length?

This one doesn't look too long

File: steps/respondent/progress-bar/RespondentProgressBar.step.js [hunk 2]
```diff
@@ -29,6 +29,19 @@ class RespondentProgressBar extends ProgressBar {
...
   }
 
+  get petitionerInferredRelationship() {
+    // eslint-disable-next-line max-len
+    const respondentInferredRelationship = this.req.session.case.data.divorceWho.toLowerCase();
+    const marriageIsSameSexCouple = this.req.session.case.data.marriageIsSameSexCouple.toLowerCase() === 'yes';
+
+    if (marriageIsSameSexCouple) {
+      return respondentInferredRelationship;
+    } else if (respondentInferredRelationship === 'husband') {
```
Comment (yarpo) [inline steps/respondent/progress-bar/RespondentProgressBar.step.js]: don't we have consts for `husband` and `wife`?

File: helpers/caseOrchestrationHelper.js (hunk 1 above)
Comment (CoffeeOwl97) [reply helpers/caseOrchestrationHelper.js]: Just checked - the following line is over 80 characters (max line length = 80 for DA)

File: steps/respondent/progress-bar/RespondentProgressBar.step.js (hunk 2 above)
Comment (CoffeeOwl97) [reply steps/respondent/progress-bar/RespondentProgressBar.step.js]: Not in DA as far as a global search can see.

**Comment by nsingamsetty** [comment]
This is tested and working as expected. Refer DIV-5373 for evidences.

**Pull request merged**